caom2utils/caom2utils/blueprints.py

```diff
@@ -1491,11 +1491,17 @@ def _guess_axis_info(self):
 
 
 def _to_float(value):
-    return float(value) if value is not None else None
+    try:
```

Comment:
This changes the whole meaning of the function and can break existing functionality (if it exists). IMO, a better way would be to have a different method `_extract_float(value)` or similar which would imply best effort.